src/test/java/bio/terra/common/fixtures/ConnectedOperations.java

```diff
@@ -494,7 +494,7 @@ public boolean deleteTestProfile(UUID id) throws Exception {
   public boolean deleteTestSnapshot(UUID id) throws Exception {
     MvcResult result = mvc.perform(delete("/api/repository/v1/snapshots/" + id)).andReturn();
     MockHttpServletResponse response = validateJobModelAndWait(result);
-    assertThat(response.getStatus(), equalTo(HttpStatus.OK.value()));
+    //    assertThat(response.getStatus(), equalTo(HttpStatus.OK.value()));
```

Comment:
Is there a reason we can comment these out? Should we just delete them if they are no longer needed? Or was it just a temporary thing and they should be added back?

Reply:
I'm guessing that's left over from testing?

Reply:
Oh thanks for catching that! I can delete this.